source/state_representation/test/tests/test_cartesian_state.cpp

```diff
@@ -68,6 +68,132 @@ TEST(CartesianStateTest, RandomWrenchInitialization) {
   EXPECT_TRUE(random.get_wrench().norm() > 0);
 }
 
+TEST(CartesianStateTest, CopyState) {
+  CartesianState state1 = CartesianState::Random("test");
+  CartesianState state2(state1);
+  EXPECT_EQ(state1.get_name(), state2.get_name());
+  EXPECT_EQ(state1.get_reference_frame(), state2.get_reference_frame());
+  EXPECT_TRUE(state1.data().isApprox(state2.data()));
+  CartesianState state3 = state1;
+  EXPECT_EQ(state1.get_name(), state3.get_name());
+  EXPECT_EQ(state1.get_reference_frame(), state3.get_reference_frame());
+  EXPECT_TRUE(state1.data().isApprox(state3.data()));
+}
+
+TEST(CartesianStateTest, CopyPose) {
+  CartesianPose pose1 = CartesianPose::Random("test");
+  CartesianPose pose2(pose1);
+  EXPECT_EQ(pose1.get_name(), pose2.get_name());
+  EXPECT_EQ(pose1.get_reference_frame(), pose2.get_reference_frame());
+  EXPECT_TRUE(pose1.data().isApprox(pose2.data()));
+  EXPECT_TRUE(pose2.get_twist().norm() == 0);
```

Comment:
I think we had this before, it would be better to use `EXPECT_EQ(pose.get_twist().norm(), 0)` because the fail message would be more verbose.

Reply:
Duh there were so many of them to change, felt like an endless task. But should be good for now... For `CartesianState` at least.

Reply:
Sorry :smile: I'm happy to do it next time hehe